packages/create-gasket-app/lib/scaffold/actions/load-template.js

```diff
@@ -0,0 +1,118 @@
+import path from 'path';
+import os from 'os';
+import { withSpinner } from '../with-spinner.js';
+import { PackageManager } from '@gasket/utils';
+import { mkdtemp, rm } from 'fs/promises';
+
+const VERSION_TAG_REGEX = /@([\^~]?\d+\.\d+\.\d+(?:-[\d\w.-]+)?|[\^~]?\d+\.\d+\.\d+|[a-zA-Z]+|file:.+)$/;
+
+/**
+ * Parses template string into name and version components
+ * @param {string} template - Template string (e.g., "@gasket/template-api@1.0.0")
+ * @returns {{ name: string, version: string }} Parsed template name and version
+ */
+function parseTemplateNameAndVersion(template) {
+  if (!VERSION_TAG_REGEX.test(template)) {
+    return { name: template, version: 'latest' };
+  }
```

Comment:
The main reason for my suggestion to make the function more concise was this gnarly regex, which seems like a scary thing to try to maintain. I bet the function will work as expected if you remove this `VERSION_TAG_REGEX` block and the regex const.